cli-access-protected-api/pom.xml

```diff
@@ -20,13 +20,13 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>msal4j</artifactId>
-      <version>1.11.0</version>
+      <version>1.11.2</version>
```

Comment:
any chance for us to use `*` in here instead?

Reply:
[Sure can](https://maven.apache.org/pom.html#Dependency_Version_Requirement_Specification). For some reason however, it's only picking up 1.11.1 and not 1.11.2.  For the record, the syntax for maven would be. 

```xml
<version>[1.11,)</version>
```

 I'll probably just leave it like what I've got for now, but thanks for the suggestion!  I have a feeling that this will get enrolled in the version update bot that other samples have (the dependency get a comment metadata tag added, and that powers some automation to handle updates).  I've seen that on other samples, and Marsh was going to look into it for these.